src/components/common/index.js

```diff
@@ -7,26 +7,29 @@ export const createDisabledTextStyles = () => css`
   color: ${({ theme }) => theme.textDisabled};
   text-shadow: 1px 1px ${({ theme }) => theme.textDisabledShadow};
   /* filter: grayscale(100%); */
+  -webkit-text-fill-color: ${({ theme }) => theme.inputTextDisabled};
 `;
 export const createBoxStyles = () => css`
-    box-sizing: border-box;
-    display: inline-block;
-    background-color: ${({ theme }) => theme.material};
-    color: ${({ theme }) => theme.text};
-  `;
+  box-sizing: border-box;
```

Comment:
ESLint was complaining quite a lot in this project and I'm not sure if this is some issue with my local setup or not 🤔 If I run `npm run lint` I'm getting quite some errors. I can revert those (auto-fix) changes below if necessary.

Reply:
@rwachtler Thanks for the info. @luizbaldi just fixed those eslint issues. Can you pull from the changes from master, auto-fix files and commit it again? thanks!

Reply:
Thanks for the info. Will do it this evening! 🙂

Reply:
@arturbien Done! ✅